lib/index.js

```diff
@@ -1,29 +1,7 @@
-/**
- * Plugins must be valid NodeJS require-able modules, usually shipped as a directory containing either:
- *  - an `index.js` file in its root directory,
- *    exporting a valid Javascript class exposing an `init` method, or
- *  - a well-formed `package.json` file in its root directory,
- *    specifying the path of the main require-able in the `main` field.
- *
- * To determine the Plugin name, Kuzzle looks for the `name` field
- * in the `package.json` file falling back to the plugin directory name otherwise.
- *
- * @see http://docs.kuzzle.io/plugins-reference/plugins-creation-prerequisites/
- */
 class CorePlugin {
```

Comment:
nitpicking: we could rename the plugin class with a more explicit name (`ComputedFiledsPlugin` for instance ?)